wikimedia/wikipedia-ios
# Update Insert Image wizard mediawiki links to consider device language

**Phabricator:** https://phabricator.wikimedia.org/T377887

### Notes
* Adds Special:MyLanguage and uselang= query items to mediawiki URLS to redirect to translated versions if available
* It falls back to English if translation is not available

### Test Steps
1. Set device language to Spanish
2. Go to the insert image wizard, and tap the learn more links
3. They should open in the Spanish translation of the page
4. Set device language to German (or other languages without translations)
5. Link should open in English
